TestsCalculator/Pages/CalculatorPage.cs

```diff
@@ -25,6 +25,8 @@ public CalculatorPage(IWebDriver driver)
         public string Income => _driver.FindElement(By.Id("income")).GetAttribute("value");
         public string Interest => _driver.FindElement(By.Id("interest")).GetAttribute("value");
         public string EndDate => _driver.FindElement(By.Id("endDate")).GetAttribute("value");
+        public IWebElement CurrencySymbol => _driver.FindElement(By.XPath("(//td [@id=\"currency\"])"));
```

Comment:
You may use  single quote symbol inside xpaths. More over, you may make this property 'string', it is good if you don't use the web-element itself, only its text.

```suggestion
        public string CurrencySymbol => _driver.FindElement(By.XPath("(//td [@id='currency'])")).Text;
```